certified-connectors/tyntec-SMS/apiDefinition.swagger.json

```diff
@@ -1,14 +1,15 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "tyntec-SMS",
-    "description": "Send a custom SMS message using the tyntec conversation API. You can also use Status_check to check status (accepted/delivered/failed)",
+    "title": "tyntec SMS Business",
```

Comment:
@ridlees We usually suggest partners to name their connectors with their product or company name, As per your product name, it should be "tyntec SMS", Can you please let us know why you would like to add"Business" to the title.

Reply:
Sure, it was a proposed name convention for all new connectors. It should follow the name convention of tyntec products (see https://www.tyntec.com/viber-business-messages, https://www.tyntec.com/whatsapp-business-api, https://www.tyntec.com/sms?country=1893&currency=1820&family=1826)